### DISCUSSION TITLE
Backup Rotate

### DISCUSSION THREAD

**Discussion by linuxgokan** [abbbi/virtnbdbackup#216] [General]
Hello

I delete backups every 7 days with "find". This backup I make won't cause any harm, right? It is automatically backed up every night using the "auto" option. It only takes incremental backups.

**Comment by abbbi** [comment]
sorry, i dont understand the question here, if you want to rotate your backups you may want to look at the following scripts implementing a backup rotation and using virtnbdbackup:

https://github.com/slackdaystudio/vm-snapshot
https://github.com/abbbi/virtnbdbackup/blob/master/scripts/vmbackup.sh

auto backup mode will create full backup if no incremental backup is existant anymore.
you however need to make sure not to remove incremental backups you would still require to create a full restore.

**Comment by linuxgokan** [comment]
If the NAS disk fills up, is it OK if I buy another NAS with a new full backup? Are checkpoints a problem?
